From 2397314542265405498ea8c82121c174ed9011a5 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 09 Oct 2013 20:09:56 +0000
Subject: test: Call g_type_init() with older glib

Only call g_type_init() when compiling against an older glib.

https://bugs.freedesktop.org/show_bug.cgi?id=70255
---
diff --git a/test-tree-magic.c b/test-tree-magic.c
index 9b4dfaf..fd5cd7e 100644
--- a/test-tree-magic.c
+++ b/test-tree-magic.c
@@ -637,7 +637,10 @@ int main (int argc, char **argv)
 	char *content, **lines;
 	guint i;
 
+#if GLIB_CHECK_VERSION(2,36,0)
+#else
 	g_type_init ();
+#endif
 
 	if (argc != 2) {
 		g_print ("Usage: %s [file]\n", argv[0]);
--
cgit v0.9.0.2-2-gbebe
